### PR TITLE
[Medical Rewrite] Add HealTime modifiers to settings

### PR DESCRIPTION
**Pull request by Arcanum417**
**When merged this pull request will:**
- Let user set absolute time for HealTime from settings framework
- Let user set absolute time for Medkit treatmentTime

Using settings framework, you can now set absolute  treatmentTime Healtime which is used for PAK. Medkit now calls healTimePartial and applies  treatmentTime which can also be set in settings framework.

### CODE REVIEW

File: addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp [hunk 1]
```diff
@@ -305,6 +305,7 @@ class GVAR(Actions) {
         items[] = {"ACE_medKit"};
         treatmentLocations[] = {QEGVAR(medical,useLocation_MedKit)};
         requiredMedic = QEGVAR(medical,medicSetting_MedKit);
+		treatmentTime = QUOTE(_target call FUNC(healTime)) * QEGVAR(medical,MedkitTimeModifier);
```
Comment (Neviothr) [inline addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp]: Needs to be just *slightly* tabbed out.

File: addons/medical_treatment/functions/fnc_healTime.sqf [hunk 2]
```diff
@@ -18,13 +18,14 @@
...
 private _totalDamage = 0;
+private _treattime = 0;
 
 {
     _totalDamage = _totalDamage + _x;
 } forEach (_unit getVariable [QEGVAR(medical,bodyPartDamage), []]);
-if (QEGVAR(medical,PAKTime) != 0) then {
-    _time = QEGVAR(medical,PAKTime);
+if (EGVAR(medical,PAKTime) > 0) then {
+    _treattime = EGVAR(medical,PAKTime);
```
Comment (Neviothr) [inline addons/medical_treatment/functions/fnc_healTime.sqf]: `_treattime` should probably be `_treatTime`. Camel case or whatever it's called.

File: addons/medical_treatment/functions/fnc_HealTimeMedkit.sqf [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+params ["_unit"];
+
+private _totalDamage = 0;
+private _treattime = 0;
+
+{
+    _totalDamage = _totalDamage + _x;
+} forEach (_unit getVariable [QEGVAR(medical,bodyPartDamage), []]);
+if (EGVAR(medical,Medkitime) > 0) then {
+    _treattime = EGVAR(medical,MedkitTime);
```
Comment (Neviothr) [inline addons/medical_treatment/functions/fnc_HealTimeMedkit.sqf]: Same here.

File: addons/medical_treatment/functions/fnc_HealTimeMedkit.sqf [hunk 4]
```diff
@@ -18,14 +18,14 @@
 params ["_unit"];
 
 private _totalDamage = 0;
-private _treattime = 0;
+private _treattTime = 0;
```
Comment (Neviothr) [inline addons/medical_treatment/functions/fnc_HealTimeMedkit.sqf]: There's an extra `t`.

**Comment by commy2** [comment]
Merge? For the sake of testing, I'd want to have it in the big PR.

**Comment by SilentSpike** [comment]
I say merge

**Pull request merged**